packages/components/src/tabs/types.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import type * as Ariakit from '@ariakit/react';
+
+/**
+ * Internal dependencies
+ */
+import type { IconType } from '../icon';
+
+export type TabsContextProps =
+	| {
+			/**
+			 * The tabStore object returned by Ariakit's `useTabStore` hook.
+			 */
+			store: Ariakit.TabStore;
+			/**
+			 * The unique id string for this instance of the Tabs component.
+			 */
+			instanceId: string;
+	  }
+	| undefined;
+
+export type TabsProps = {
+	/**
+	 * The children elements, which should be at least a
+	 * `Tabs.Tablist` component and a series of `Tabs.TabPanel`
+	 * components.
+	 */
+	children: React.ReactNode;
+	/**
+	 * When `true`, the tab will be selected when receiving focus (automatic tab
+	 * activation). When `false`, the tab will be selected only when clicked
+	 * (manual tab activation). See the official W3C docs for more info.
+	 * .
+	 *
+	 * @default true
+	 *
+	 * @see https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/
+	 */
+	selectOnMove?: boolean;
+	/**
+	 * The id of the tab to be selected upon mounting of component.
+	 * If this prop is not set, the first tab will be selected by default.
+	 * The id provided will be internally prefixed with the
+	 * `TabsContextProps.instanceId`.
+	 *
+	 * Note: this prop will be overridden by the `selectedTabId` prop if it is
+	 * provided. (Controlled Mode)
+	 */
+	initialTabId?: string;
+	/**
+	 * The function called when a tab has been selected.
+	 * It is passed the `instanceId`-prefixed `tabId` as an argument.
```

Comment:
I thought we were stripping the instance ID away, keeping it as a implementation detail internal to the component?

Reply:
You are right, we are! I missed updating this description when that change was made, thank you.